templates/web/isleofwight/report/_council_sent_info.html

```diff
@@ -0,0 +1,17 @@
+[% SET duration_clause = problem.duration_string(c) %]
+[% IF duration_clause || problem.can_display_external_id %]
```

Comment:
Doesn't need to call this function here if it has its own template, I guess. Shrug. Does it matter if it still says council on .com, as it currently does.